build.gradle.kts

```diff
@@ -42,9 +31,9 @@ teavm.js {
     targetFileName = "fruzhin.js"
 }
 
-//teavm {
-//    js {
-//        sourceMap.set(true)
-//        debugInformation.set(true)
-//    }
-//}
\ No newline at end of file
+teavm {
+    js {
+        sourceMap.set(true)
```

Comment:
couldn't we merge this with teavm.js above?

Reply:
The sourceMap and debugInformation will be removed later on